index.ts

```diff
@@ -64,5 +65,21 @@ export const claimsMetadata: ClaimsMetadataSummary = {
         country: "schema:addressCountry"
       }
     ]
+  },
+  businessCard: {
+    type: ['Credential', 'ProofOfBusinessCardCredential'],
```

Comment:
I think this makes more sense following the example of the `publicProfile`, so type shouldn't include `ProofOf`